### ISSUE TITLE
Chisel issue on Xcode 7.3

### ISSUE THREAD

**Issue by syjiangrui** [facebook/chisel#149]
Xcode 7.3 and iOS 9.3 simulator.
OS:OSX 10.11.4
pviews and presponder doesn't work and no output.
pvc works well.

**Comment by kastiglione** [comment]
Thanks for the report. I will look into this in a couple days.

**Comment by Jowyer** [comment]
`pjson` does not work either

**Comment by nickjackson2011** [comment]
the same problem, import module failed.
waiting for resolving~

**Comment by ppaulojr** [comment]
Same problem here. Many commands like pviews, wivar don't display any text.

**Comment by kastiglione** [comment]
Thanks for everyone weighing in. This is on my list to investigate, I'll try to get to it as soon as I can!

**Comment by kastiglione** [comment]
Looks like the commands are being run, but the results aren't being printed. I'll look into how stdout is configured.

**Comment by kastiglione** [comment]
On my machine, when I run lldb's built-in `script` command with no arguments, the interactive interpreter is broken in Xcode 7.3. Can anyone confirm this? With previous versions of Xcode, running `script` would give this output:

```
Python Interactive Interpreter. To exit, type 'quit()', 'exit()'.
>>> 
```

and python code could be evaluated from the `>>>` prompt. But this doesn't work for me with Xcode 7.3.

**Comment by SourceChang** [comment]
![7 2 1](https://cloud.githubusercontent.com/assets/15812732/14098593/3b4c7750-f5b0-11e5-8f62-7c4871b89ec3.png)
![7 3](https://cloud.githubusercontent.com/assets/15812732/14098594/3d7e8d42-f5b0-11e5-9700-c5a7b46ff094.png)

Xcode 7.2.1 was still work
hope this can help you

**Comment by syjiangrui** [comment]
The built-in `script` command doesn't work in my Xcode 7.3 either. @kastiglione

**Comment by kastiglione** [comment]
Thanks, @syjiangrui. I'll dive a little deeper to see if there's a workaround. I'll file a radar too.

**Comment by kastiglione** [comment]
I filed a radar for this, number 25415928 (rdar://25415928).

**Comment by bcattle** [comment]
In the meantime a workaround is to run `po [[[UIApplication sharedApplication] keyWindow] recursiveDescription]` from the debugger

**Comment by Dzionis** [comment]
`pcurl` does not work either

**Comment by palcalde** [comment]
Inside Resolved Issue of Xcode 7.3 release, they mention an issue with Python that was resolved. Weird.

https://developer.apple.com/library/ios/releasenotes/DeveloperTools/RN-Xcode/Chapters/xc7_release_notes.html#//apple_ref/doc/uid/TP40001051-CH5-SW1

![image](https://cloud.githubusercontent.com/assets/1252619/14203848/9a5f0cb0-f800-11e5-8414-4691c2593289.png)

**Comment by kastiglione** [comment]
Thanks for the link. The Known Issues section confirms my hypothesis:

> LLDB Python interpreter is not able to perform console I/O within Xcode. As a result, the script command hangs when it would normally prompt for input and printed output from Python scripts does not appear in the Xcode debug console. (24877720)

I'm amazed they knowingly shipped this.

**Comment by EdgarHz** [comment]
mac os version 10.11.3
xcode version 7.3(7d175)
pviews print nothing.

I use: **print >> result, description**   replaced: **print  description**  then **pviews -d 3** is ok
I'm newly to lldb and python, I hope I can take some help for you.

just like this:
```
--- a/fblldb.py
+++ b/fblldb.py
@@ -12,6 +12,7 @@ import lldb
+import commands
@@ -76,7 +77,8 @@ def makeRunCommand(command, filename):
     if validateArgsForCommand(args, command):
-      command.run(args, options)
+      command.run(args, options,result)

@@ -88,7 +88,7 @@ class FBPrintViewHierarchyCommand(fb.FBCommand):
         prefixToRemove = separator * maxDepth + " "
         description += "\n"
         description = re.sub(r'%s.*\n' % (prefixToRemove), r'', description)
-      print description
+      print >> result, description
```

**Comment by sunbohong** [comment]
https://github.com/sunbohong/chisel
临时解决方案。
a temporary situation。

**Comment by qsw1214** [comment]
@sunbohong  not work

**Comment by sunbohong** [comment]
@qsw1214  i only changed the `pviews`.Maybe you should uninstall your old chisel.

**Comment by palcalde** [comment]
@kastiglione I've tested today with new seed Xcode 7.3.1 beta, and, even tho they mention they have fixed the Python Interpreter.. it doesn't work. 
![image](https://cloud.githubusercontent.com/assets/1252619/14681643/4d3bdd56-0722-11e6-805b-833c755355fa.png)

**Comment by kastiglione** [comment]
thanks @palcalde! I just came here to say the release notes say that the issue was fixed, but I haven't tried yet, glad we have confirmation.

## tl;dr Upgrade to Xcode 7.3.1

**Comment by bcattle** [comment]
Didn't he say it "didn't work"?

**Comment by yas375** [comment]
I also was confused by same thing today when I received email notifications from Github. But turns our @palcalde had edited his comment on GitHub to an opposite.

![2016-04-20_1352](https://cloud.githubusercontent.com/assets/217368/14690115/584913f8-06ff-11e6-8b5a-fbb2e75a9c55.png)

**Comment by bcattle** [comment]
Oh hahahahaha I'm sorry. That's funny. Sorry to waste your time and thanks
for the explanation. 👍

**Comment by palcalde** [comment]
Sorry guys you are correct! At the beginning I thought it wasn't working but I relized it was me messing up the different Xcode versions I have installed. Then I edited the message. It works! :)

**Comment by Ambtion** [comment]
I use xcode 7.3 (7d)，and pviews print nothing
